src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
+        this.radius = radius;
+        super.color = color;

```

Comment:
The correct way to pass a value to a superclass constructor in Java is by calling `super` as the first statement in the subclass constructor.